src/Changelog/BumpAndCommitChangelogVersionInterface.php

```diff
@@ -8,7 +8,7 @@
 use Laminas\AutomaticReleases\Git\Value\SemVerVersion;
 use Laminas\AutomaticReleases\Gpg\SecretKeyId;
 
-interface BumpAndCommitChangelogVersion
+interface BumpAndCommitChangelogVersionInterface
```

Comment:
Ooof, no, this is a net loss.

`laminas/laminas-coding-standard` is really a no-go, if it enforces this stuff.